README.md

```diff
@@ -1,42 +1,76 @@
   <a href="https://github.com/coinbase/rosetta-ethereum/blob/master/LICENSE.txt"><img src="https://img.shields.io/github/license/coinbase/rosetta-ethereum.svg" /></a>
   <a href="https://pkg.go.dev/github.com/coinbase/rosetta-ethereum?tab=overview"><img src="https://img.shields.io/badge/go.dev-reference-007d9c?logo=go&logoColor=white&style=shield" /></a>
 </p>
+<!-- Rosetta Tagline -->
+<p align="center">
+Build once.
+Integrate your blockchain everywhere.
+</p>
+<!-- Release cycle information -->
 
 <p align="center"><b>
 ROSETTA-ETHEREUM IS CONSIDERED <a href="https://en.wikipedia.org/wiki/Software_release_life_cycle#Alpha">ALPHA SOFTWARE</a>.
-USE AT YOUR OWN RISK! COINBASE ASSUMES NO RESPONSIBILITY OR LIABILITY IF THERE IS A BUG IN THIS IMPLEMENTATION.
+USE AT YOUR OWN RISK.
 </b></p>
 
+<!-- Short License Info -->
+<p align="center">This project is available open source under the terms of the [Apache 2.0 License](https://opensource.org/licenses/Apache-2.0).</p>
+
+<!-- Overview -->
 ## Overview
-`rosetta-ethereum` provides a reference implementation of the Rosetta API for Ethereum in Golang. If you haven't heard of the Rosetta API, you can find more information [here](https://rosetta-api.org).
 
-## Features
-* Comprehensive tracking of all ETH balance changes
-* Stateless, offline, curve-based transaction construction (with address checksum validation)
-* Atomic balance lookups using go-ethereum's GraphQL Endpoint
-* Idempotent access to all transaction traces and receipts
+The `rosetta-ethereum` repository provides an implementation sample of the Rosetta API for Ethereum in Golang. We created this repository for developers of Ethereum-like (a.k.a., account-based) blockchains, who may find it easier to fork this implementation sample than write one from scratch.
+
+[Rosetta](https://www.rosetta-api.org/docs/welcome.html) is an open-source specification and set of tools that makes integrating with blockchains simpler, faster, and more reliable. The Rosetta API is specified in the [OpenAPI 3.0 format](https://www.openapis.org).
+
+You can craft requests and responses with auto-generated code using [Swagger Codegen](https://swagger.io/tools/swagger-codegen) or [OpenAPI Generator](https://openapi-generator.tech). These requests and responses must be human-readable (easy to debug and understand), and able to be used in servers and browsers.
+
+Jump to:
 
-## System Requirements
-`rosetta-ethereum` has been tested on an [AWS c5.2xlarge instance](https://aws.amazon.com/ec2/instance-types/c5).
-This instance type has 8 vCPU and 16 GB of RAM. If you use a computer with less than 16 GB of RAM,
-it is possible that `rosetta-ethereum` will exit with an OOM error.
+* [How to Use This Repo](#How-to-Use-This-Repo)
+* [Docker Deployment](#Docker-Deployment)
+* [Testing](#Testing)
+* [Documentation](#Documentation)
+* [Related Projects](#Related-Projects)
+<!-- h2 How to Use This Repo -->
+## How to Use This Repo
+
+1. Finish reading this README file.
```

Comment:
Do we really need to mention this?

Reply:
You'd be surprised!

Reply:
tbh - nobody will read the full readme before starting even if we mention this. I am pretty sure I'd not do it. 😅  I have never read the whole readme of any project before starting. 🤷

Reply:
I'll fix it in the next commit.